package.json

```diff
@@ -64,5 +64,5 @@
       "eslint --cache --fix"
     ]
   },
-  "packageManager": "yarn@3.2.3"
+  "packageManager": "yarn@1.22.22"
```

Comment:
I wonder why we need to downgrade yarn.

Reply:
I had trouble with yarn on my local and set the yarn version to the one I had. I will revert this.